xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -111,21 +111,21 @@
                                                       "xbmc_epg_types.h" \
                                                       "libXBMC_pvr.h"
 
-#define ADDON_INSTANCE_VERSION_SCREENSAVER            "1.0.0"
-#define ADDON_INSTANCE_VERSION_SCREENSAVER_MIN        "1.0.0"
+#define ADDON_INSTANCE_VERSION_SCREENSAVER            "1.0.1"
+#define ADDON_INSTANCE_VERSION_SCREENSAVER_MIN        "1.0.1"
 #define ADDON_INSTANCE_VERSION_SCREENSAVER_XML_ID     "kodi.binary.instance.screensaver"
 #define ADDON_INSTANCE_VERSION_SCREENSAVER_DEPENDS    "xbmc_scr_dll.h" \
                                                       "xbmc_scr_types.h"
 
-#define ADDON_INSTANCE_VERSION_VFS                    "1.0.0"
-#define ADDON_INSTANCE_VERSION_VFS_MIN                "1.0.0"
+#define ADDON_INSTANCE_VERSION_VFS                    "1.0.1"
+#define ADDON_INSTANCE_VERSION_VFS_MIN                "1.0.1"
 #define ADDON_INSTANCE_VERSION_VFS_XML_ID             "kodi.binary.instance.vfs"
 #define ADDON_INSTANCE_VERSION_VFS_DEPENDS            "kodi_vfs_dll.h" \
                                                       "kodi_vfs_types.h" \
                                                       "kodi_vfs_utils.hpp"
 
-#define ADDON_INSTANCE_VERSION_VISUALIZATION          "1.0.0"
-#define ADDON_INSTANCE_VERSION_VISUALIZATION_MIN      "1.0.0"
+#define ADDON_INSTANCE_VERSION_VISUALIZATION          "1.0.1"
+#define ADDON_INSTANCE_VERSION_VISUALIZATION_MIN      "1.0.1"
 #define ADDON_INSTANCE_VERSION_VISUALIZATION_XML_ID   "kodi.binary.instance.visualization"
 #define ADDON_INSTANCE_VERSION_VISUALIZATION_DEPENDS  "xbmc_vis_dll.h" \
```

Comment:
Not sure, but shouldn't the version numbers for the other add-on types changed in this PR (audio encoder etc.) also be bumped?